### PR TITLE
Add documentation for the addon option of the EmberApp constructor.

### PR DESCRIPTION
**Pull request by kfoisy**

### CODE REVIEW

File: _posts/2013-04-08-managing-dependencies.md [hunk 1]
```diff
@@ -156,6 +156,21 @@ _Note: The built-in assets are required dependencies needed by the environment
 to run your app. If you use the above method to specifically exclude
 some, you should still be including them in some other way._
 
+##### Whitelisting and Blacklisting Assets
+
+You can limit which dependencies in your package.json file get imported into your Ember application by using the the addon option of the EmberApp constructor. A whitelist parameter allows you to restrict modules to a specific list. A blacklist parameter excludes specific modules from being imported into your app:
```
Comment (kellyselden) [inline _posts/2013-04-08-managing-dependencies.md]: "the the"

**Review by kellyselden — DISMISSED** [review]
small typo, then LGTM

File: _posts/2013-04-08-managing-dependencies.md (hunk 1 above)
Comment (kfoisy) [reply _posts/2013-04-08-managing-dependencies.md]: Should be fixed now.

**Comment by nathanhammond** [comment]
LGTM! Thanks for your attention to detail, @kfoisy!

**Pull request merged**